src/goto-symex/symex_goto.cpp

```diff
@@ -82,10 +82,8 @@ void goto_symext::symex_goto(statet &state)
       frame.loop_iterations[goto_programt::loop_id(*state.source.pc)].count;
     unwind++;
 
-    // continue unwinding?
```

Comment:
I would recommend squashing with the prior commit. Simple test: would reverting either of these commits in isolation be meaningful? I don't think so.